recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,23 @@
+PACKAGECONFIG_append = " networkd resolved"
```

Comment:
The meta-mender-demo layer already adds this kind of stuff.  @kacf and I chatted briefly about splitting that up to have multiple sub-layers.  Should we try to do that first? This is definitely duplicating code; although given the timing maybe we live with that now and create a JIRA task to clean it up later.

Reply:
I vote for the second option, since the split is not entirely trivial, and requires some planning.